Makefile

```diff
@@ -1,122 +1,92 @@
-tag := latest
-git_hash ?= $(shell git log -1 --pretty=format:%h)
-
-base_dir ?= $(shell git rev-parse --show-toplevel)
-examples ?=  ../$(base_dir)/examples
-app_dir ?= $(shell git rev-parse --show-prefix)
-
-CONDA_EXE := conda
-CONDA_ENV ?= $(base_dir)/env
-env := $(CONDA_ENV)
-conda_run := $(CONDA_EXE) run -p $(env)
-PYTEST_EXE := $(CONDA_ENV)/bin/pytest
-
 MIN_NODE_VER := 14
-MIN_NPM_VER := 6
-NODE_VER    := $(shell node -v | cut -d. -f1 | sed 's/^v\(.*\)/\1/')
-NPM_VER     := $(shell npm -v | cut -d. -f1)
-
-ifeq ($(shell uname -s), Darwin)
-    SED_I_ARG := -i ''
-else
-    SED_I_ARG := -i
-endif
+MIN_NPM_VER  := 6
+MIN_PY3_VER  := 8
+NODE_VER     := $(shell node -v | cut -d. -f1 | sed 's/^v\(.*\)/\1/')
+NPM_VER      := $(shell npm -v | cut -d. -f1)
+PY3_VER      := $(shell python3 -c "import sys;t='{v[1]}'.format(v=list(sys.version_info[:2]));print(t)")
+PY_OK        := $(shell python3 -c "print(int($(PY3_VER) >= $(MIN_PY3_VER)))")
+
+all:
+	@echo "\nThere is no default Makefile target right now. Try:\n"
+	@echo "make setup - check your environment and install the dependencies."
+	@echo "make clean - clean up auto-generated assets."
+	@echo "make build - build PyScript."
+	@echo "make precommit-check - run the precommit checks (run eslint)."
+	@echo "make test-integration - run all integration tests sequentially."
+	@echo "make fmt - format the code."
+	@echo "make fmt-check - check the code formatting.\n"
 
 .PHONY: check-node
 check-node:
 	@if [ $(NODE_VER) -lt $(MIN_NODE_VER) ]; then \
-		echo "Build requires Node $(MIN_NODE_VER).x or higher: $(NODE_VER) detected"; \
+		echo "\033[0;31mBuild requires Node $(MIN_NODE_VER).x or higher: $(NODE_VER) detected.\033[0m"; \
 		false; \
 	fi
 
 .PHONY: check-npm
 check-npm:
 	@if [ $(NPM_VER) -lt $(MIN_NPM_VER) ]; then \
-		echo "Build requires Node $(MIN_NPM_VER).x or higher: $(NPM_VER) detected"; \
+		echo "\033[0;31mBuild requires Node $(MIN_NPM_VER).x or higher: $(NPM_VER) detected.\033[0m"; \
 		false; \
 	fi
 
-setup: check-node check-npm
+.PHONY: check-python
+check-python:
+	@if [ $(PY_OK) -eq 0 ]; then \
+		echo "\033[0;31mRequires Python 3.$(MIN_PY3_VER).x or higher: 3.$(PY3_VER) detected.\033[0m"; \
+		false; \
+	fi
+
+# Check the environment, install the dependencies.
+setup: check-node check-npm check-python
```

Comment:
Hi @JeffersGlass - good question and one that I agonised over. 

On the one hand, for absolute beginners, that might be quite useful.

But on the other non-beginners or tech folk won't have too much of a problem understanding both the concept and that there are many ways to skin that cat (in the docs [see here](https://github.com/pyscript/docs/pull/35), using virtualenvmanager, pyenv, the IDE, whatever). 

So I assumed technical know how and the choice to configure that aspect of one's environment oneself.

Like I said, happy to be persuaded otherwise.